.github/workflows/end2end.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       && github.event.issue.state == 'open'
       && contains(github.event.comment.body, '!test')
       && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
+      && github.ref != 'refs/heads/main'
```

Comment:
Have you tested this works? I'm worried this will never return `true`, because `issue_comment` actions are always triggered from the default branch and we haven't checked out the base branch of the PR at this point.

Reply:
I'm also afraid of this scenario.

Another way of fetching informations on concerned branch might be to use [`github.event.issue.pull_request`](https://docs.github.com/en/webhooks/webhook-events-and-payloads#issue_comment) to get info on the source branch.

Reply:
@tchataigner I've managed to find a workaround in https://github.com/lurk-lab/solidity-verifier/pull/45. A nifty alternative is to run `gh pr view <pr-number> --json baseRefName --jq '.baseRefName'`, but GitHub certainly doesn't make this easy to figure out :smile:

Reply:
You are both right. We have https://github.com/lurk-lab/solidity-verifier/pull/45 for tackling with this, and probably we will need merge it first

Reply:
https://github.com/lurk-lab/solidity-verifier/pull/45 has been merged and it looks like we have this issue resolved